dev-tools/global.d.ts

```diff
@@ -0,0 +1,29 @@
+import * as BattleType from './../sim/battle'
+import * as DataType from './../sim/dex-data'
+import * as DexType from './../sim/dex'
+import * as PokemonType from './../sim/pokemon'
+import * as PRNGType from './../sim/prng'
+import * as SideType from './../sim/side'
+
+declare global {
+	// modules
+	const Dex: typeof DexType
```

Comment:
The type name is `ModdedDex`; `Dex` is reserved for the gen7 instance.

Reply:
<code>Dex</code> here refers to <code>global.Dex</code>, but I added the <code>ModdedDex</code> name below.